egs/librispeech/ASR/conformer_ctc/README.md

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
This is how a pre-trained model can be used to transcribe a sound file.
@danpovey 

It depends on

- torchaudio, for reading sound files
- [kaldifeat](https://github.com/csukuangfj/kaldifeat), for feature extraction

Reply:
Only HLG decoding with the transformer encoder output is added.
Do we need to use the attention decoder for rescoring?

Reply:
Also, do we need to support transcribe multiple files at the same time?
It's easy to support that.

Reply:
This is great-- thanks!
Regarding using the attention decoder for rescoring-- yes, I'd like you to add that,  because this will probably
be a main feature of the tutorial, and I think having good results is probably worthwhile.